components/library/libidn2/libidn2.p5m

```diff
@@ -26,14 +26,13 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 file path=usr/bin/$(MACH64)/idn2
 file path=usr/bin/idn2
 file path=usr/include/idn2/idn2.h
-#file path=usr/lib/$(MACH64)/charset.alias
-link path=usr/lib/$(MACH64)/libidn2.so target=libidn2.so.0.3.5
-link path=usr/lib/$(MACH64)/libidn2.so.0 target=libidn2.so.0.3.5
-file path=usr/lib/$(MACH64)/libidn2.so.0.3.5
+link path=usr/lib/$(MACH64)/libidn2.so target=libidn2.so.0.3.6
+link path=usr/lib/$(MACH64)/libidn2.so.0 target=libidn2.so.0.3.6
+file path=usr/lib/$(MACH64)/libidn2.so.0.3.6
 file path=usr/lib/$(MACH64)/pkgconfig/libidn2.pc
-link path=usr/lib/libidn2.so target=libidn2.so.0.3.5
-link path=usr/lib/libidn2.so.0 target=libidn2.so.0.3.5
-file path=usr/lib/libidn2.so.0.3.5
+link path=usr/lib/libidn2.so target=libidn2.so.0.3.6
+link path=usr/lib/libidn2.so.0 target=libidn2.so.0.3.6
+file path=usr/lib/libidn2.so.0.3.6
```

Comment:
What is its SONAME?

Reply:
```
$ objdump -p /usr/lib/libidn2.so.0.3.6 | grep SONAME
  SONAME               libidn2.so.0
```
Why? Don't like it being zero?

Reply:
I'm just asking because ABI tracker mentioned that they have occasionally increased SONAME last time.

Reply:
Yes, they botched it last time. We may wait for 2.2.0 to appear in the ABI Tracker to be sure about this and other similar problems.

Reply:
ABI Tracker is for libidn2 2.2.0 is published and is clean.